source/includes/container_dev_env.inc.rst

```diff
@@ -1,8 +1,13 @@
 .. note::
 
-   The rsyslog project maintains
-   `rsyslog docker development environment images <https://hub.docker.com/u/rsyslog/>`_.
```

Comment:
I really think we should have the hyperlink in, and so with proper text. This also has todo with getting google juice over the non-official containers... Is it a problem to do the replacement inside a link? Let's discuss concerns.

Reply:
>  Is it a problem to do the replacement inside a link?

I think there is a technical limitation. I recall reading this again recently, but I don't remember where off the top of my head. Let me look for it and I'll respond back here with a reference (providing I can find it).

Reply:
While I think we can pull off a replacement inside a link, it does not appear to be as straightforward as it would need to be. The easiest fix here is just to revert to the original link. 

My goal was to put the focus on the rsyslog Docker Hub itself and not the development images. If you feel the focus should be on the images instead, we could alter the subst reference and then rework the other references. This would put the focus on the images instead of the Docker Hub (which I saw as a landing page to learn more about available images).

Reply:
Valid points. I guess that text was already a bit outdated by now. Let's nail this PR and go for the next one.